tests/zenko_tests/node_tests/s3SDK.js

```diff
@@ -50,4 +50,23 @@ const ringS3Client = new S3({
     httpOptions: { timeout: 0 },
 });
 
-module.exports = { scalityS3Client, awsS3Client, ringS3Client };
+const altScalityS3Client = new S3({
```

Comment:
I'm not quite sure how this client works, are the `AWS_ACCESS_KEY` and `AWS_SECRET_KEY` credentials injected into Zenko?

Reply:
Yeah, they're set as secrets on every run, I believe. Basically what I needed though was another client pointing to Zenko but with different credentials

Reply:
hm, I guess I'm confused because I assume that these AWS credentials would only for the AWS account used in the e2e test